content/courses/tokens-and-nfts/token-program.md

```diff
@@ -485,31 +488,40 @@ import {
 } from "@solana-developers/helpers";
 import { Connection, clusterApiUrl } from "@solana/web3.js";
 
-const connection = new Connection(clusterApiUrl("devnet"));
+async function main() {
```

Comment:
No need for async wrappers, this is mentioned very explicitly in CONTRIBUTING.md. Fix this everywhere.